packages/react/src/patterns/sections/LeadSpace/__tests__/LeadSpaceImage.test.js

```diff
@@ -7,21 +7,21 @@ describe('<LeadSpaceImage />', () => {
     const image = [
       {
         minWidth: 0,
-        url: 'https://picsum.photos/id/1076/320/370',
+        url: 'https://dummyimage.com/320x370/ee5396/161616',
       },
       {
         minWidth: 672,
-        url: 'https://picsum.photos/id/1076/672/400',
+        url: 'https://dummyimage.com/672x400/ee5396/161616',
       },
       {
         minWidth: 1056,
-        url: 'https://picsum.photos/id/1076/1056/480',
+        url: 'https://dummyimage.com/1056x480/ee5396/161616',
       },
     ];
 
     const imageInfo = {
-      default: 'https://picsum.photos/id/1076/1056/480',
-      alt: 'lead space image',
+      default: 'https://dummyimage.com/1056x480/ee5396/161616',
```

Comment:
Are these missing an aspect ratio?

Reply:
Possibly, @oliviaflory @larahanlon2 what are the aspect ratios for the LeadSpace imagery?

Reply:
I left these out intentionally because I don't know the rules for aspect ratio in the leadspace and didn't want to force anything. So I kept the image dimensions that were being used. @annawen1 @wonilsuhibm, is there an aspect ratio for leadspace images?

Reply:
@kennylam you don't need to update this specific file or any `LeadSpaceImage` file...it won't be used for the LeadSpace anymore, we're switching over to the `Image` component instead

Reply:
@kennylam but also, dimensions are here:

```

| Name      | Data Type | Description                                                 |
| --------- | --------- | ----------------------------------------------------------- |
| `mobile`  | String    | Image with aspect ratio (320 x 370) for mobile breakpoint   |
| `tablet`  | String    | Image with aspect ratio (672 x 400) for tablet breakpoint   |
| `default` | String    | Image with aspect ratio (1056 x 480) for desktop breakpoint |
| `alt`     | String    | Alt description of the image                                |
```

Reply:
@kennylam LeadSpace was merged in, so this should be moot.